MetaMask/metamask-extension
# Update "Add Snap account" registry schema

## **Description**

This PR changes the snap detail schema to match the schema used in in the snaps registry. 

Depends on: https://github.com/MetaMask/keyring-snaps-registry/pull/6

## **Manual testing steps**

1. Click on account picker
2. Click on `Add account or hardware wallet`
3. Click on `Add snap account`
4. See the snaps in the list

## **Related issues**

Resolves https://github.com/MetaMask/accounts-planning/issues/97

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
